src/load_csv.py

```diff
@@ -9,7 +9,7 @@
 from dialog.models.db import session
 
 
-def load_csv_and_generate_embeddings(path, cleardb=False):
+def load_csv_and_generate_embeddings(path, cleardb=False, columns=["content"]):
```

Comment:
```suggestion
def load_csv_and_generate_embeddings(path, cleardb=False, columns=("content",)):
```

Reply:
@walison17 why are we fallbacking to a tuple here?